settings/Search.setting.php

```diff
@@ -211,4 +211,22 @@
     'description' => 'If enabled, only primary details (eg contact\'s primary email, phone, etc) will be included in Basic and Advanced Search results. Disabling this feature will allow users to match contacts using any email, phone etc detail.',
     'help_text' => NULL,
   ),
+  'quicksearch_options' => array(
+    'group_name' => 'Search Preferences',
+    'group' => 'Search Preferences',
+    'name' => 'quicksearch_options',
+    'type' => 'Array',
```

Comment:
I think this should actually be 'string' & we should also have a 'serialize' => Json key

Reply:
Why?